<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="myKeyInput">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>280</y>
      <width>211</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextEdit" name="myTextEditInput">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>110</y>
      <width>241</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Bản rõ</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>320</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Bản mã</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>250</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>K</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="myTextEditOutput">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>350</y>
      <width>241</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="swap">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>500</y>
      <width>111</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Chuyển </string>
    </property>
   </widget>
   <widget class="QPushButton" name="openPlaintText">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>70</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>File</string>
    </property>
   </widget>
   <widget class="QPushButton" name="openCypherText">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>70</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>File</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>80</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Bản mã</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>320</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Bản rõ</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="myTextEditInput2">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>110</y>
      <width>241</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="myTextEditOutput2">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>350</y>
      <width>241</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>351</width>
      <height>511</height>
     </rect>
    </property>
    <property name="title">
     <string>Mã hóa</string>
    </property>
    <widget class="QPushButton" name="save1">
     <property name="geometry">
      <rect>
       <x>162</x>
       <y>470</y>
       <width>111</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Lưu</string>
     </property>
    </widget>
    <widget class="QPushButton" name="myEncryptButton">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>250</y>
       <width>83</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Mã hóa</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Reset">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>83</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Reset</string>
     </property>
    </widget>
    <widget class="QPushButton" name="autoKey">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>220</y>
       <width>83</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Tạo khóa</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>30</y>
      <width>361</width>
      <height>511</height>
     </rect>
    </property>
    <property name="title">
     <string>Giải mã</string>
    </property>
    <widget class="QPushButton" name="save2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>470</y>
       <width>121</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Lưu</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Reset2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>83</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Reset</string>
     </property>
    </widget>
    <widget class="QPushButton" name="myDecryptButton">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>250</y>
       <width>81</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Giải mã</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="myKeyInput_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>250</y>
       <width>211</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>220</y>
       <width>63</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>K</string>
     </property>
    </widget>
   </widget>
   <zorder>groupBox</zorder>
   <zorder>groupBox_2</zorder>
   <zorder>myKeyInput</zorder>
   <zorder>myTextEditInput</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>label_5</zorder>
   <zorder>myTextEditOutput</zorder>
   <zorder>swap</zorder>
   <zorder>openPlaintText</zorder>
   <zorder>openCypherText</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>myTextEditInput2</zorder>
   <zorder>myTextEditOutput2</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
